### PR TITLE
Added section explaining the Action type and Control type settings

### PR DESCRIPTION
**Pull request by duckets**
### Description

Improve documentation around action type and control type selection
https://jira.unity3d.com/browse/ISX-1834

### Changes made

Added a section describing action type and control type in the Actions Editor page.

### CODE REVIEW

File: Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md]: Could we list some common ones here: 
E.g. suggest Vector2 as a common selection for gamepad stick input, or for use with a composite WASD input

I'm still not sure why I would use 2D axis over Vector2 or when I would use dpad

2dvector and dpad are mentioned here - with no clarity over why using one over the other : https://docs.unity3d.com/Packages/com.unity.inputsystem@1.7/manual/ActionBindings.html#2d-vector
Could/should we list other types on this page too?

This page shows dpad but not why its used over 2d-vector
https://docs.unity3d.com/Packages/com.unity.inputsystem@1.8/manual/Actions.html#creating-actions-in-code

Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md]: add extra information that Axis or Dpad offers extra processing over 2d vector
Axis adds dead zone and clamping.
Dpad changes the interface to up/down/left/right digital response.

File: Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md
Comment (ekcoh) [inline Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md:60]: Nitpick but "mouse click" is very different from "mouse button press". The former is an interaction and the second is a button press which this relates to. Hence recommend "mouse buttons" here or similar.

Comment (ekcoh) [reply Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md:60]: Also a mouse click typically also relates not only to button but also to movement which may discard it as a click.

File: Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md
Comment (ekcoh) [inline Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md:62]: I leave it to you but could use the term "interaction phases" here since that what started and canceled is.

**Review by ekcoh — APPROVED** [review]
LGTM, good improvement @duckets. Some minor comments I leave to you to decide to update so will approve already now.

File: Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md:72]: I'd quite like to see a like here saying the most common selections for value types are:
* Bool for a digital button (although Button is usually used for this)
* Analog for an trigger button with analog values
* Axis for a one dimensional axis (such as mouse horizonal movement 
* Vector2 for a stick, mouse coordinate or composite input such as WASD,

File: Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md:74]: I think you could also mention that these more specific control types have a runtime class that can be accessed in the action binding callback context that makes it more descriptive when accessing apsects such as up,down,left,right on the dpad

 void Start()
    {
        actionDpad.performed += ctx => OnDpadPress(ctx.control as DpadControl);

        actionDpad.Enable();
    }

    void OnDpadPress(DpadControl dpad)
    {
        if (dpad.down.isPressed)
            Debug.Log("DPad down");
    }

**Review by lyndon-unity — APPROVED** [review]
Its a good step forward but I added some further recommendations.

File: Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md
Comment (ekcoh) [reply Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md:74]: Lets land this without this change @duckets

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/InputSystem?pullRequest=1841) <br/>All committers have signed the CLA.

**Pull request merged**
